docs/moosedocs.py

```diff
@@ -2,10 +2,15 @@
 import sys
 import os
 import pdb
+import re
 
 # Locate MOOSE directory
 MOOSE_DIR = os.getenv('MOOSE_DIR', os.path.join(os.getcwd(), '..', 'moose'))
 if not os.path.exists(MOOSE_DIR):
+  RE = re.compile(r'(.*?/moose)(.*)')
+  matches = RE.search(os.getcwd())
+  MOOSE_DIR = matches.group(1)
+if not os.path.exists(MOOSE_DIR):
```

Comment:
I think the logic for this might be wrong, but I am not sure yours is quite right either. I will fix this up, can you just remove changes to this file from this PR, it is a different problem.